packages/exemplar-storybook-react/package.json

```diff
@@ -35,7 +35,7 @@
     "css-loader": "^3.2.0",
     "diagnostics": "^1.1.1",
     "node-sass": "^4.13.0",
-    "sass-loader": "^8.0.0",
+    "sass-loader": "^7.3.1",
```

Comment:
why does this need downgraded to `7.x.x`? I understand our particular usecase of it, but this feels like a breaking change if someone else is already using this and expecting sass-loader@8 (they are). could we make this a dev-dep + peerdep on `^7.0.0 | ^8.0.0` since we're not really configuring it here anyway?